apparmor.d/abstractions/chromium

```diff
@@ -93,26 +102,34 @@
   /usr/share/hwdata/pnp.ids r,
   /usr/share/mozilla/extensions/{,**} r,
   /usr/share/qt{5,}/translations/*.qm r,
+  /usr/share/uim/* r,
   /usr/share/webext/{,**} r,
 
   /etc/@{name}/{,**} r,
   /etc/fstab r,
-  /etc/igfx_user_feature{,_next}.txt w,
+  /etc/igfx_user_feature{,_next}.txt rw,
   /etc/opensc.conf r,
 
   /var/lib/dbus/machine-id r,
   /etc/machine-id r,
 
+  /var/lib/uim/* r,
+  /var/lib/uim/installed-modules.scm r,
```

Comment:
`/var/lib/uim/installed-modules.scm r,` is already covered by `/var/lib/uim/* r,`